Packs/Cybereason/pack_metadata.json

```diff
@@ -2,7 +2,7 @@
     "name": "Cybereason",
     "description": "Endpoint detection and response to manage and query malops, connections and processes.",
     "support": "partner",
-    "currentVersion": "2.1.5",
```

Comment:
Let's make this a revision since the fix is very small. To do this you can manually edit this file to 2.1.6, and manually rename the `2_2_0.md` release notes file to `2_1_6.md`.

Reply:
Done